BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemProcessor.swift

```diff
@@ -261,7 +261,9 @@ final class AddEditItemProcessor: StateProcessor<// swiftlint:disable:this type_
             let ownershipOptions = try await services.vaultRepository
                 .fetchCipherOwnershipOptions(includePersonal: !isPersonalOwnershipDisabled)
 
-            state.collections = try await services.vaultRepository.fetchCollections(includeReadOnly: false)
+            // We need read-only collections so that we can include them in the state
+            // to correctly calculate if the item can be deleted
+            state.collections = try await services.vaultRepository.fetchCollections(includeReadOnly: true)
```

Comment:
👍 The comment here is helpful context!

⚠️ I believe we were only fetching the non-read only collections because we don't want to show read-only collections as an option where you can assign the cipher to that collection in the ownership section. Do we need to add an additional filter for only showing ones that you have permissions for?

<img width="568" alt="Screenshot 2024-12-04 at 10 02 45 AM" src="https://github.com/user-attachments/assets/5ad865c6-1f08-4cae-8845-812eff586efd">

Reply:
🤦🏼‍♀️ I totally thought of this at some point this morning when I was making tea and then forgot when I got back to my computer. Yes, that is a thing that should be done. I'll make the fix.